newIDE/app/src/MainFrame/EditorTabs/DropIndicator.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import * as React from 'react';
+import ThemeConsumer from '../../UI/Theme/ThemeConsumer';
+
+const styles = {
+  columnDropIndicator: {
+    borderRight: '1px solid #18dcf2',
+    borderLeft: '1px solid #18dcf2',
+    width: 7,
+    marginLeft: '-1px',
+    height: '100%',
+    boxSizing: 'border-box',
+  },
+};
+
+export function ColumnDropIndicator() {
+  return (
+    <ThemeConsumer>
+      {gdevelopTheme => (
```

Comment:
See component `ToolbarSeparator` to use the context that allows you to access the theme from the function. This way you won't have to use this `#18dcf2`